internal/operators/ocs/validations.go

```diff
@@ -12,13 +12,13 @@ import (
 
 type Config struct {
 	OCSMinimumCPUCount                int64             `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"18"`
-	OCSMinimumRAMGB                   int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
+	OCSMinimumRAMGiB                  int64             `envconfig:"OCS_MINIMUM_RAM_COUNT" default:"57"`
```

Comment:
Please update `OCS_MINIMUM_RAM_COUNT` to `OCS_MINIMUM_RAM_GiB`

Reply:
Done